assets/schema_input.json

```diff
@@ -12,13 +12,17 @@
                 "pattern": "^\\S+$",
                 "uniqueItems": true,
                 "errorMessage": "Sequence reference name must be provided and cannot contain spaces",
-                "meta": ["id"]
+                "meta": [
+                    "id"
+                ]
             },
             "taxid": {
                 "type": "integer",
-                "uniqueItems": true,
+                "uniqueItems": false,
```

Comment:
```suggestion
```

Reply:
Yeah that was me throwing stuff at the wall when it wouldn't validate